python/langsmith/evaluation/_arunner.py

```diff
@@ -472,14 +476,8 @@ async def _aevaluate(
         description=description,
         num_repetitions=num_repetitions,
         runs=runs,
-        include_attachments=_include_attachments(target)
-        or _evaluators_include_attachments(evaluators) > 0,
-        reuse_attachments=num_repetitions
-        * (
-            int(_include_attachments(target))
-            + _evaluators_include_attachments(evaluators)
-        )
-        > 1,
+        include_attachments=num_include_attachments > 0,
+        reuse_attachments=num_repetitions * num_include_attachments > 1,
```

Comment:
I know it's already there but I have to think a lot about what the condition is testing for here, can we add a comment or preferably simplify this code to be readable at a glance?